source/Applications/Advanced/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -139,6 +154,12 @@ int main()
     {
         Zivid::Application zivid;
 
+        // camera settings for both 2D and 3D acquisitions set
+        const std::chrono::microseconds exposureTime = std::chrono::microseconds{ 20000 };
+        const double aperture = 5.66;
+        const double gain = 1.0;
+        const double brightness = 1.0;
```

Comment:
Declare and initialize variables just before using them, meaning, move them just before:

`const auto bgr = (use2D ? getImage2D(camera, makeSettings2D(exposureTime, aperture, gain, brightness))
                                : getImage3D(camera, makeSettings(exposureTime, aperture, gain, brightness)));

`